src/tic-tac-toe.ts

```diff
@@ -69,12 +69,13 @@ class Game {
   }
 
   private checkGameForWin(): void {
-    const winCondition =
-      this.checkRowsForWin() ||
-      this.checkColumnsForWin() ||
-      this.checkDiagonalsForWin();
+    const winCondition = [
+      this.checkRowsForWin(),
+      this.checkColumnsForWin(),
+      this.checkDiagonalsForWin()
+    ];
 
-    if (winCondition) {
+    if (winCondition.some(winnerCheck => winnerCheck === true)) {
```

Comment:
Since the function passed to `some` expects a boolean to be returned, and the values we're iterating _are_ booleans, this function could just be the [identity function](https://en.wikipedia.org/wiki/Identity_function): 
`winConditions.some(condition => condition)`

Reply:
Great! I wasn't positive if that was the case, but knew it was weird to write `=== true`. Good to know